_docs/overview/key-concepts.md

```diff
@@ -13,14 +13,13 @@ padlocked box, of which there can be multiple padlocks - see the
 [Multi-signature Security](/docs/overview/multi-signature-security)
  section.
 
-Unlike a password, a private key is not meant for you to remember.
-It's a long string of gibberish.
-The private key is what you need to keep
-secure. If anyone gets it, they can take your money. Unlike traditional
-financial instruments, there is no recourse. There is no company that is
-liable, because Bitcoin is a decentralized system not run by any person or
-entity. And no law enforcement agency is likely to investigate your
-case.
+Unlike a password, a private key is not meant to be memorized. It is a
+256-bit number, usually expressed as a 64-character hexadecimal string.
+It is important to understand that the holder of the private key is the
+owner of the bitcoin and, unlike traditional financial instruments, 
```

Comment:
Holding the private key gives one access to the bitcoins but does not make that person the owner. If a thief steals my key, I am still the owner, and he is still a thief.

Reply:
nm, I see you addressed this in a later commit already